dockers/docker-base-stretch/Dockerfile.j2

```diff
@@ -53,7 +53,11 @@ RUN apt-get update &&        \
         libjemalloc1         \
         liblua5.1-0          \
         lua-bitop            \
-        lua-cjson
+        lua-cjson            \
+# For DHCP Monitor tool
+        libexplain-dev       \
+        libexplain51         \
+        libpcap-dev
```

Comment:
From https://packages.debian.org/stretch/libpcap-dev
Empty package to facilitate upgrades, can be safely removed.

Reply:
Good catch! will replace with libpcap-dev with libpcap0.8 as it is required during runtime. I was under the impression that libpcap-dev would pull the latest

Reply:
There should be no need to install dev packages in the final image; only packages needed for runtime.